packages/adblocker-puppeteer/adblocker.ts

```diff
@@ -277,6 +279,9 @@ export class PuppeteerBlocker extends FiltersEngine {
     } while (true);
   };
 
+  public setRequestInterceptionPriority = (defaultPriority = 0) =>
+    (this.priority = defaultPriority);
+
   public onRequest = (details: puppeteer.HTTPRequest): void => {
```

Comment:
If you do update this, you might consider waiting for this upcoming PR https://github.com/puppeteer/puppeteer/pull/7796/files#diff-9eddf4dc51bf6a0125ca7fb094468ad284112270385c41cebce4f8f0a29620abR5035 because it introduces a legacy mode guard too:

```ts
if (details.isInterceptResolutionHandled()) return
```

Reply:
Note: here also is a backward compatibility check that is safer:

`if (details.isInterceptResolutionHandled?.()) return`

Reply:
@benallfree I created #2343 to gather all the changes in one PR.